honeycombio/example-greeting-service
# feat: dockerize dotnet services

## Which problem is this PR solving?

- services are easier and more predictable to run in docker

## Short description of the changes

- use dockerized dotnes services in tilt
- supersedes #334 

## To verify
- In repo root
- set up env vars
- `tilt up dotnet`
- curl greeting, check trace
- `tilt down`
- `tilt up` another language should still work
